dashboard/app/models/pd/teacher_progress_report.rb

```diff
@@ -0,0 +1,56 @@
+class Pd::TeacherProgressReport
+
+  def self.generate_report_for_user(user)
+    teachers =
+      if user.admin?
+        # All teachers in all workshops
+        Pd::Attendance.distinct_teachers
```

Comment:
That's a lower priority work item, for after facilitator summit. We plan to add a date filter to all the reports and probably pagination as well.